### ISSUE TITLE
Improve error messaging on go fmt err

### ISSUE THREAD

**Issue by sanbornm** [hooklift/gowsdl#40]
Current message shows something like this:

🍀  75:53: expected type, found ',' (and 2 more errors)

It should mention something about not a valid WSDL or unable to parse schema.

**Comment by c4milo** [comment]
Agreed. Although, ideally we should validate Basic Profile 1.0 on any WSDL before attempting to generate code. But, I haven't really had time to do that.
